cards_exporter/cards_pdf_exporter.py

```diff
@@ -0,0 +1,32 @@
+import pdfkit
```

Comment:
no documantion, those function are really hard to understand for someone who doesn't know the libraries